### DISCUSSION TITLE
showing just one of more parts

### DISCUSSION THREAD

**Discussion by EricaMadams** [w3c-cg/musicxml#447] [General]
Hi I‘m verry new to coding but have a little bit of experience in html. I want to display some sheetmusic. My question is:

Lets say I have 3 instruments in a score.

Is there a way to tell the code that I just want Part one (P1) to be displayed? Or just part 3 NOT to be displayed?
I tought to comment out the Pt1 but it still shows the (empty) system.

**Comment by adrianholovaty** [comment]
Hi @EricaMadams! Generally this is something handled by the specific music notation software you're using, as opposed to being encoded in the MusicXML file itself. But, yes, if you want to do it in the MusicXML document, you could comment out (or delete) the appropriate `<part>` elements and `<score-part>` (plus likely some other things, depending on the nature of the notation).
